texting/twilio.py

```diff
@@ -230,8 +239,14 @@ def chain_sms_async(
 
     task = get_task_for_function(send_sms)
     tasks: List[Any] = []
+    media_substr = "media_url="
     for body in bodies:
-        sig = task.si(phone_number, body)
+        # bit jank but unless i rework the whole chaining system theres not a good way ?
+        if body.find(media_substr):
```

Comment:
I think this needs to be `if body.find(media_substr) >=0:`, since the pattern could be at the beginning of the string and return `0` then not run the block, and if the string doesn't appear it returns `-1` and (news to me!) that is truthy and it does run the block. This second case is what I think is causing the tests to fail on CI

Reply:
oo good catch about the `.find` returning 0. 
also didn't think -1 would evaluate to truthy because i'm perpetually in javascript land. 

ended up using `in` instead because it's cleaner